docs/_docs/localization/index.md

```diff
@@ -7,20 +7,26 @@ nav_order: 12
 # Localization
 
 We are developing tools for developers to localize the figma design into different languages more easily.
+
 The first introduced tool is a figma plugin that generates a strings xml file containing the strings
 used by the figma design.
 
+The second tool is a figma plugin that literates all the images, let user assign a proper name and
+download selected images as a `drawable.zip` file.
```

Comment:
Suggestion: "The second tool is a figma plugin that iterates through all the images, then lets a user assign a proper name and download selected images as a `drawable.zip` file."